upup/pkg/fi/cloudup/vsphere/vsphere_cloud.go

```diff
@@ -36,6 +40,18 @@ func (c *VSphereCloud) ProviderID() fi.CloudProviderID {
 	return fi.CloudProviderVSphere
 }
 
+func NewVSphereCloud(spec *kops.ClusterSpec) (*VSphereCloud, error) {
+	server := spec.VSphereServer
+	datacenter := spec.VSphereDatacenter
+	cluster := spec.VSphereResourcePool
+	username := os.Getenv("VSPHERE_USERNAME")
+	password := os.Getenv("VSPHERE_PASSWORD")
+
```

Comment:
Ok, let's make sure that when we make use of these values, user does get relevant message.